webapp/components/create_comment.jsx

```diff
@@ -500,6 +573,20 @@ export default class CreateComment extends React.Component {
             addButtonClass += ' disabled';
         }
 
+        let emojiPicker = null;
+        if (this.state.showEmojiPicker) {
+            //TODO: get custom emojis
```

Comment:
We shouldn't have any TODOs in code, we should make tickets for them. If this isn't going in for this PR @jasonblais can you work with @bonespiked to make a ticket for this? Then we can remove this TODO